app/utils/product_data.py

```diff
@@ -27,15 +31,22 @@ def generateProductData(data_params: ProductDataParams, db_session: Session) ->
 
     print(period_start_date, period_end_date)
 
-    data = (
+    """data = (
```

Comment:
Could you add a comment to explain what this is doing? I'm assuming it's along the lines of creating a data frame from SQL query, but am slightly confused by the `"""` syntax.